dcrjson/dcrwalletextcmds.go

```diff
@@ -124,6 +124,23 @@ type GetTicketsCmd struct {
 	IncludeImmature bool
 }
 
+// NewGetTicketsCmd creates a new GetTicketsCmd.
```

Comment:
// NewGetTicketsCmd returns a new instance which can be used to issue a gettickets JSON-RPC command.